### PR TITLE
Init all repos for org

### PR DESCRIPTION
**Pull request by lubalexuan**
## JIRA
[Link to JIRA](insert url)

## Overview
- [ ] Bumped up the version in VERSION file

### CODE REVIEW

File: cmd/init.go [hunk 1]
```diff
@@ -47,11 +48,23 @@ See https://help.github.com/articles/searching-code/ for more details about the
...
+To init all repos belonging to a specific org use --all-repos flag. Caution: you should run this command first to update all your local github repos.
+
+- cd into folder with all repos belonging to an org
+
+- for all repositories pull remote changes from github
+
+$ find . -type d -depth 1 -exec git --git-dir={}/.git --work-tree=$PWD/{} pull origin master \;
 
-$ mp init "org:Clever" --repo-search
+After that you can run the following command.
```
Comment (rgarcia) [inline cmd/init.go]: why is it necessary to pull your local git repos? AFAICT --all-repos using the github API, and then from there further `microplane` operations do a fresh clone.

Comment (lubalexuan) [reply cmd/init.go]: I think `clone` was cloning the local version of github repos. As a result microplane planned changes for a number of repos that should't have been included.

Comment (rgarcia) [reply cmd/init.go]: I don't see how that's possible... the call to `git clone` that microplane makes is always a git URL https://github.com/Clever/microplane/blob/937d048f6b63bf73927f9ffc713d6bb196315361/clone/clone.go#L35

Comment (lubalexuan) [reply cmd/init.go]: Hm weird. I will remove this note from the documentation.

I _think_ I ran into issues before without pulling changes from github. I worked on this a while ago so can't be sure anymore.

Comment (taylor-sutton) [reply cmd/init.go]: If you don't remove your `mp/` dir between runs and re-run `mp` in the same directory, it'll see the old `mp/repo-name/clone` and whatnot. (I sometimes use this intentionally so I don't have to reclone, but it's often not what you want). Is that what happened?

**Comment by lubalexuan** [comment]
> Is this supposed to produce a different result than
> `mp init --repo-sarch "org:Clever"`? Or just a more convenient/reliable way to do that?

I tried to run the above command before. --repo-search doesn't seem to work properly, and this was a quick fix 🤷‍♀

**Pull request merged**